### PR TITLE
Bug fix/loading state

### PR DESCRIPTION
**Pull request by enickgoku**
Created loading symbol upon user and image load.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ import './App.css'
 import FaceRecognition from './components/FaceRecognition'
 import GitHub from './components/GitHub/GitHub'
 import { handleLogout } from './utils/session'
+import Loading from './components/Loading'
```
Comment (jonbelaire) [inline src/App.js]: Please cluster all of your imports by root directory at the top of each file.

```
react & other packages

./utils

./components

./Component.module.css
```

Separate each cluster with a new line.

File: src/components/Navigation/index.js [hunk 2]
```diff
@@ -11,9 +11,7 @@ const Navigation = () => {
       <nav style={{ display: 'flex', justifyContent: 'flex-end' }}>
         <p
           onClick={() => {
-            localStorage.removeItem('userId')
-            localStorage.removeItem('token')
-            navigate('../signin')
+            handleLogout()
```
Comment (jonbelaire) [inline src/components/Navigation/index.js]: This doesn't need to be an anonymous function. Simply pass `handleLogout` as the prop value.

File: src/components/SignIn/index.js [hunk 3]
```diff
@@ -12,6 +12,12 @@ function SignIn() {
   const [formData, setFormData] = useState(initialState)
   const [error, setError] = useState(false)
 
+  useEffect(() => {
+    const userId = localStorage.getItem('userId')
+    const token = localStorage.getItem('token')
+    if (userId && token) navigate('/')
+  }, [navigate])
```
Comment (jonbelaire) [inline src/components/SignIn/index.js:19]: We should create a function that determines whether or not a user has a `token` and `userId` by returning a boolean value that can be used in conditional statements.

Comment (jonbelaire) [reply src/components/SignIn/index.js:19]: We can introduce this in a different PR (issue https://github.com/enickgoku/facerecognitionbrain/issues/2)

Comment (jeriale) [reply src/components/SignIn/index.js:19]: https://github.com/enickgoku/facerecognitionbrain/issues/2

**Pull request merged**